api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -107,6 +110,12 @@ public void validate(Object target, Errors errors) {
 				validateStopDatetime(visit, otherVisit, errors);
 			}
 		}
+		
+		// Check visit not before patient's birthdate
+		if (visit.getPatient() != null) {
+			validateBirthDate(visit, errors);
```

Comment:
how about renaming the private method you added to `validateVisitStartedBeforePatientBirthdate`, this lets you remove your comment as its clear from the method name what it does.
and move the `visit.getPatient() != null` inside the private method.